dataland-community-manager/src/main/resources/application-nodb.properties

```diff
@@ -8,4 +8,6 @@ spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
 dataland.apikeymanager.base-url=https://local-dev.dataland.com/api-keys
 dataland.authorization.publiclinks=/actuator/health,/actuator/health/ping,/actuator/info,/swagger-ui/**,/v3/api-docs/**
 dataland.backend.base-url=https://local-dev.dataland.com/api
-server.port=8081
\ No newline at end of file
+server.port=8081
+
+spring.flyway.enabled=false
```

Comment:
Why is this necessary?

Reply:
Is it because now its the first time we have flyway for this service?

Reply:
@fsoffel You've worked out the migration, do you know?

Reply:
We have this done for all services that use flyway so far. The reason for this is that the nodb profile is mostly used for unit tests (@SpringApplicationTest or something like that basically boots some part of the application and if it cannot connect to the for flyway specified database it fails) and other things like open api docs generation which runs independent and fails if it can not connect to the for flyway specified database. Thus since we do not need it for these purposes anyways usually, flyway got disabled.

Reply:
Thank you for the explanation!